pom.xml

```diff
@@ -2,14 +2,14 @@
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>com.xebialabs.jira.addons</groupId>
+    <groupId>org.venkatk.jira.addons</groupId>
```

Comment:
If you would like to publish to Atlassian Marketplace under you own name, please fork this repository and make the changes on your fork.  Then you can rename and perform all the changes you need to maintain your version on the Atlassian Marketplace.

No name changes should be made to this repository.